src/test/java/seedu/address/ui/testutil/GuiTestAssert.java

```diff
@@ -71,4 +77,64 @@ public static void assertListSize(PersonListPanelHandle personListPanelHandle, i
     public static void assertResultMessage(ResultDisplayHandle resultDisplayHandle, String expected) {
         assertEquals(expected, resultDisplayHandle.getText());
     }
+
+    /**
+     * Returns the color style for {@code tagName}'s label. The tag's color is determined by looking up the color
+     * in {@code PersonCard#TAG_COLOR_STYLES}, using an index generated by the hash code of the tag's content.
+     */
+    private static String getTagColorStyleFor(String tagName) {
+        switch (tagName) {
+
+        case "classmates":
+            return "pink";
+
+        case "owesMoney":
+            return "lightBlue";
+
+        case "colleagues":
+            return "deepOrange";
+
+        case "neighbours":
+            return "gray";
+
+        case "family":
+            return "red";
+
+        case "friend":
+            return "amber";
+
+        case "friends":
+            return "lightBlue";
+
+        case "husband":
+            return "gray";
+
+        default:
+            throw new AssertionError(tagName + " does not have a color assigned.");
+        }
+    }
+
+    /**
+     * Asserts that the tags in {@code actualCard} matches all the tags in {@code expectedPerson} with the correct
+     * color.
+     */
+    private static void assertTagsEqual(Person expectedPerson, PersonCardHandle actualCard) {
```

Comment:
This method name might be confusing because it's actually also checking if colours match too.

Reply:
Changed method name to `assertTagsAndTagColorStylesEqual` to be clearer